### ISSUE TITLE
can you share data please ?

### ISSUE THREAD

**Issue by Sandy4321** [CRIPAC-DIG/SR-GNN#38]
great code thanks

I try to learn it 
but this link for data is broken

can you share data please ?

**Comment by SXKDZ** [comment]
You may download digentica [here](https://sr-gnn.bj.bcebos.com/train-item-views.csv).

**Comment by Sandy4321** [comment]
thank you very much 
only one question

as I can see user id often is NA
is it correct?

![image](https://user-images.githubusercontent.com/11426119/85475326-eecd5a00-b583-11ea-8a95-5b49588dec29.png)

**Comment by Sandy4321** [comment]
also it is train data
is test data existing  ?

**Comment by SXKDZ** [comment]
Yes, it's correct since we consider anonymous recommendation only.
We build training set and test set according to time intervals. Please refer to our paper for details.
